src/components/Header.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+type HeaderProps = {
+  inputRef: React.Ref<HTMLInputElement>;
+  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+};
+
+export const Header = ({ inputRef, onSubmit }: HeaderProps) => {
+  return (
+    <header className="todoapp__header">
+      {/* this button should have `active` class only if all todos are completed */}
+      <button
+        type="button"
+        className="todoapp__toggle-all active"
```

Comment:
As the comment above says, this button should have `active` class only if all todos are completed